packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -90,14 +88,6 @@ function Export({
             type='password'
             value={pass}
           />
-          {pass.length < MIN_LENGTH && pass !== '' && (
```

Comment:
Doesn't really make sense to show password validation errors during actual usage of a password (as opposed to during creation).